### DISCUSSION TITLE
Set cookies with path in loaders

### DISCUSSION THREAD

**Discussion by shanmugharajk** [remix-run/remix#9710] [Q&A] [answered]
```typescript
export const loader = () => {
  // code here

  return json(
    {},
    {
      headers: [
        [
          'Set-Cookie',
          `routeNested=routeNestedValue; domain=${domain}; path='/'; httpOnly=true; secure=true;`,
        ],
      ],
    },
  );
};
```

Though I'm setting the cookies with path as `/` still it's not setting the value I'm passing in the loader.  

For example if the 
- route  `route.nested.tsx`
- url      `http://localhost:3000/route/nested`

then the cookie is set with `/route`.

Is this expected or I'm doing something wrong here. Could anyone please guide me how to achieve this?

**Comment by shanmugharajk** [comment]
Sure, that's a copy / paste mistake here sorry for that. Even without these `httpOnly=true; secure=true;` also the same behaviour

**Comment by sergiodxa** [comment, marked as answer]
I saw that both in MDN and in my apps the attributes you add to a cookie like path are all capitalized, like Path=/ also without `'`, have you tried that?

**Reply by shanmugharajk** [reply]
Thanks @sergiodxa , without `'` is the key. It's working now.
